mobile/ios/simulator/ios-simulator-log-provider.ts

```diff
@@ -1,21 +1,27 @@
 import { ChildProcess } from "child_process";
 
 export class IOSSimulatorLogProvider implements Mobile.IiOSSimulatorLogProvider {
-	private isStarted: boolean;
+	private map: IDictionary<boolean> = {};
```

Comment:
this could be named better, for example simulatorsLoggingEnabled
The usage will be:
```TypeScript
this.simulatorsLoggingEnabled[deviceIdentifier] = true;
```